### ISSUE TITLE
Faulty import in SslSettingsPartDriver 

### ISSUE THREAD

**Issue by mvarblow** [OrchardCMS/Orchard#5274]
The current Import implementation in SslSettingsPartDriver seems to clear (revert to the defaults) all SSL settings when I import a recipe file that includes a Settings section but not an SslSettingsPart section.  It should leave the SSL settings alone in this case.  

The Import and Export methods in SslSettingsPartDriver seem to be unnecessary and could just be deleted.  I'll submit a pull request with this fix.

**Comment by mvarblow** [comment]
The problem exists in 1.9.x.  Should I have tagged this issue somehow to indicate that?  I don't see a way to do that.

**Comment by mvarblow** [comment]
The problem exists in 1.9.x.  Should I have tagged this issue somehow to indicate that?

**Comment by mvarblow** [comment]
The pull request is here: https://github.com/mvarblow/Orchard/pull/1

**Comment by sebastienros** [comment]
Your pull request is pointing your own repository. Please target Orchard instead ;)

**Comment by mvarblow** [comment]
Oops!  Let me try again.  https://github.com/OrchardCMS/Orchard/pull/5278
